google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -585,6 +660,17 @@ void RunAll(std::vector<std::string> const& argv) {
   std::cout << "\nRunning GetUniformBucketLevelAccess() example" << std::endl;
   GetUniformBucketLevelAccess(client, {bucket_name});
 
+  std::cout << "\nRunning SetPublicAccessPreventionEnforced() example"
+            << std::endl;
+  SetPublicAccessPreventionEnforced(client, {bucket_name});
```

Comment:
So it looks like these tests don't check that the metadata has been updated correctly when re-fetched. Probably fine?

Reply:
This is an example, not a test.  As long as the example runs I am happy.

Reply:
Sounds good.